eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/ReplicatedCassandra.scala

```diff
@@ -293,6 +293,7 @@ private[journal] object ReplicatedCassandra {
                         timestamp: Instant,
                         deleteTo: DeleteTo,
                         origin: Option[Origin],
+                        setSeqNr: Option[SeqNr],
```

Comment:
Could you please provide case when
```
setSeqNr.get > deleteTo.value // if setSeqNr is Some
```

Reply:
before this PR, we did clamp-down:
```scala
val deleteTo1 = deleteTo
  .value
  .min(seqNr1) // val seqNr1    = journalHead.deleteTo.fold { journalHead.seqNr } { _.value.max(journalHead.seqNr) }
  .toDeleteTo
```
in above snippet (inlined from original code), we clamp down `DeleteTo` to `min(seqNr, max(current_delete_to, seqNr))`

when "head" has `delete_to = 384` and `seqNr = 574` and we request `delete_to = 575`, we we were clamping it down to `574 = min(574, max(575, 574))`

Reply:
for new code, there is a unit-test case at https://github.com/evolution-gaming/kafka-journal/pull/676/files#diff-29aefcb72c7b9fba4c21d3237b491940e36217107a2b98c2ed8a3d2c4674c7b0R973 - we know that `Batch` would discard `append 575`, if in same poll we get `deleteTo 575` - `append 575` never happens, "head" stays at `seqNr 574`